liota/lib/transports/mqtt.py

```diff
@@ -337,18 +380,16 @@ def disconnect(self):
             ten_ms_count += 1
             time.sleep(0.01)
```

Comment:
What's this weird delay loop about? Please add a comment.

Reply:
@Venkat2811  Can you please provide comment on it?

I think it is basically to complete the disconnect.